base_layer/wallet/src/text_message_service.rs

```diff
@@ -651,7 +660,7 @@ mod test {
         assert_eq!(tms.get_screen_name(), Some("Fred".to_string()));
 
         for c in contacts.iter() {
-            tms.add_contact(c.clone())
+            let _ = tms.add_contact(c.clone());
```

Comment:
Shouldn't this be unwrapped rather than squelched?

Reply:
nope, in this test there will be no other peer to response so will always fail and this is just a crud test. This will go away in the refactor in any case.

Reply:
ACK ACK